README.md

```diff
@@ -501,4 +38,8 @@ There are many ways to contribute, from writing tutorials or blog posts, improvi
 ## Feature Requests And Bugs
 
 Please file feature requests and bugs at the [issue tracker](https://github.com/Tezsure/Dartez/issues/new). If you want to contribute to this libary, please submit a Pull Request.<br>
-You can get in touch with us for any open discussion and support via [Telegram](https://t.me/tezster).
\ No newline at end of file
+You can get in touch with us for any open discussion and support via [Telegram](https://t.me/tezster).
+
+## TODO
```

Comment:
remove to do list from read me instead add Extra tab like:
Extra
We’re building a lot of exciting features which will be released soon, So stay tuned with our updates and releases!

keep developing